packages/core/src/amazonqGumby/chat/controller/controller.ts

```diff
@@ -304,7 +305,7 @@ export class GumbyController {
 
     private async transformationFinished(message: { tabID: string; jobStatus: string }) {
         this.sessionStorage.getSession().conversationState = ConversationState.IDLE
-        this.messenger.sendJobSubmittedMessage(message.tabID, true)
```

Comment:
This was a bug. It wasn't showing in the chat because the message immediately below gets sent right after which masks the first one, but this shouldn't be showing in `transformationFinished` -- this already shows when you start a job.